src/test/java/it/clients/symphony/api/DatafeedClientV2Test.java

```diff
@@ -0,0 +1,111 @@
+package it.clients.symphony.api;
+
+import clients.symphony.api.DatafeedClient;
+import clients.symphony.api.DatafeedClientV2;
+import clients.symphony.api.constants.AgentConstants;
+import it.commons.BotTest;
+import model.DatafeedEvent;
+import model.StringId;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class DatafeedClientV2Test extends BotTest {
+
+    private DatafeedClient datafeedClient;
+
+    @Before
+    public void initClient() {
+        datafeedClient = new DatafeedClientV2(symBotClient);
+    }
+
+    @Test
+    public void listDatafeedIdsSuccess() {
+        stubGet(AgentConstants.LISTDATAFEEDV2,
+                "[\n" +
+                "    {\n" +
+                "        \"id\": \"2c2e8bb339c5da5711b55e32ba7c4687_f\"\n" +
+                "    },\n" +
+                "    {\n" +
+                "        \"id\": \"4dd10564ef289e053cc59b2092080c3b_f\"\n" +
+                "    },\n" +
+                "    {\n" +
+                "        \"id\": \"83b69942b56288a14d8625ca2c85f264_f\"\n" +
+                "    }\n" +
+                "]");
+
+        List<StringId> datafeedIds = datafeedClient.listDatafeedId();
+
+        assertNotNull(datafeedIds);
+        assertEquals("2c2e8bb339c5da5711b55e32ba7c4687_f", datafeedIds.get(0).getId());
+        assertEquals("4dd10564ef289e053cc59b2092080c3b_f", datafeedIds.get(1).getId());
+        assertEquals("83b69942b56288a14d8625ca2c85f264_f", datafeedIds.get(2).getId());
+    }
+
+    @Test
+    public void createDatafeedSuccess() {
+        stubPost(AgentConstants.CREATEDATAFEEDV2,
+                "{\n" +
+                        "    \"id\": \"21449143d35a86461e254d28697214b4_f\"\n" +
+                        "}");
+        String datafeedId = datafeedClient.createDatafeed();
+
+        assertEquals("21449143d35a86461e254d28697214b4_f", datafeedId);
+    }
+
+    @Test
+    public void readDatafeedEventSuccess() {
+        stubPost(AgentConstants.READDATAFEEDV2.replace("{id}", "21449143d35a86461e254d28697214b4_f"),
+                "{\n" +
```

Comment:
I'd prefer to see those payloads in proper `.json` files as part of the test resources